src/main/resources/application.yml

```diff
@@ -45,3 +48,6 @@ azure:
   activedirectory:
     client-id: ${AZURE_AD_API_CLIENT_ID:8fb0f9ea-6351-4251-bcca-85e298bda8e7}
     app-id-uri: ${AZURE_AD_API_ID_URI:api://8fb0f9ea-6351-4251-bcca-85e298bda8e7}
+
+govnotify:
+  api-key: ${GOVNOTIFY_API_KEY}
```

Comment:
I created a test API key named "Integration Testing" in Gov Notify.  See the [Gov Notify docs](https://docs.notifications.service.gov.uk/java.html#other-testing) on integration testing.  Messages sent using this test API key give realistic responses, so if something's wrong with the template configuration for example, the tests should fail.

Also added as a secret in the password manager.